.circleci/config.yml

```diff
@@ -107,37 +107,27 @@ jobs:
         default: 'postgres'
 
 workflows:
-  "Run specs on supported Solidus versions":
+  "Run specs on development Solidus version":
     jobs:
       - run-specs:
           context: slack-secrets
-          name: run-specs-with-solidus-3-2-rails-7-ruby-3-1-postgres
+          name: run-specs-with-postgres
 
       - run-specs:
           context: slack-secrets
-          name: run-specs-but-with-solidus-latest
-          solidus_branch: 'master'
+          name: run-specs-with-mysql
```

Comment:
But we don't know how many versions of ruby will be supported together, now they are three. Is that a constant? Maybe yes, because they are released once per year.

Reply:
Yeah, surely not blocking, it's something I've done in the past and worked well, your call 👍